src/Microsoft.AspNetCore.Html.Abstractions/HtmlString.cs

```diff
@@ -10,18 +10,23 @@ namespace Microsoft.AspNetCore.Html
     /// <summary>
     /// An <see cref="IHtmlContent"/> implementation that wraps an HTML encoded <see cref="string"/>.
     /// </summary>
-    public class HtmlEncodedString : IHtmlContent
+    public class HtmlString : IHtmlContent
     {
         /// <summary>
-        /// An <see cref="IHtmlContent"/> instance for <see cref="Environment.NewLine"/>.
+        /// An <see cref="HtmlString"/> instance for <see cref="Environment.NewLine"/>.
         /// </summary>
-        public static readonly IHtmlContent NewLine = new HtmlEncodedString(Environment.NewLine);
+        public static readonly HtmlString NewLine = new HtmlString(Environment.NewLine);
```

Comment:
- Why is the type of this field changing? Agree it has this type but not sure the change is worthwhile.
- Same question for the `Empty` type.